sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -675,6 +675,7 @@ case class DenseRank(children: Seq[Expression]) extends RankLike {
   override val aggBufferAttributes = rank +: orderAttrs
   override val initialValues = zero +: orderInit
   override def sql: String = "DENSE_RANK()"
```

Comment:
why not override the `sql` method in the `RankLike` super class? That saves a line of code :).